packages/sdk/src/modules/FusePools.ts

```diff
@@ -59,12 +60,9 @@ export function withFusePools<TBase extends FuseBaseConstructor>(Base: TBase) {
       for (let i = 0; i < assets.length; i++) {
         const asset = assets[i];
 
+        const isBorrowPaused: boolean = await comptrollerContract.callStatic.borrowGuardianPaused(asset.cToken);
```

Comment:
Why the refactor? 
We are creating a promise array to run all of these requests in parallel. 
Having this one separate will just halt the rest of the execution till it's done.

And we don't need the value to proceed, nothing depends on it. I would prefer the previous version to this.

The rest of this function is also using this style, why do it differently for `comptrollerContract.callStatic.borrowGuardianPaused` ? :shrug:

Reply:
Never mind, it is used. Just seeing it now. 🤦